index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Ernest-Kell Tribute WebPage</title>
+</head>
+<body>
+  <main id="main">
+  <h1 id="title">Dr Ernest Kell</h1>
+  <div id="img">
+    <img id="image" src="EK.jpeg" alt="Dr Ernest Kell is Elon musks dad"/>
+   <figcaption id="img-caption" > Dr Ernest Kell is a renowned alien, who wins at life, because code matters.  </figcaption>
+    </div>
+
+<section id="Tribute-info">
+    <h3>Here is a time line of Dr Ernest Kell's life!</h3>
+</section id ="headline">
+
+<ul>
+    <li>
+        <strong>1942</strong> WAVES BORN: Baby Dr Ernest Kell born via waves.
+    </li>
+    <li>
+        <strong>1998</strong> WAVES SURFED; Grown Dr spends time surfing waves.
+    </li>
+    <li>
+      <strong>2024</strong> WAVES CREATED: Wise Dr expands waves via creation.
+    </li>
+    <li>
+      <strong>2042</strong> WAVES LEARNT: Dr learns to become the wave. 
+    </li>
+    <li>
+        <strong>2099</strong> WAVES ENJOYED: Dr sits and enjoys all waves.
+    </li>
+</ul>
+
+<blockquote cite="https://opensea.io/" >
+    <p>
+     Waves, collaborating melodies, synergetic symphonies of mutual means.   
+    </p>
+    <cite>--  Wavesss of code</cite>
+</blockquote>
+
+<a id="tribute-link" target="_blank" href="https://hicetnunc.art/">ART HERE</a>
+
+
+
+
+
+</main>   
+</body>
+</html>
```

Comment:
- _(optional)_ Great job with the UI. I just recommend improving the indentation. This is because a well-indented code makes it easier to read for you and for your collaborators. It also makes your code look professional 😎

Reply:
Ok thank you! @m05e5